tests/ctst/.eslintrc.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
             },
         }],
         'linebreak-style': [1, 'unix'],
-        'new-cap': [1, { newIsCap: true }],
+        'new-cap': 'off',
```

Comment:
I would like to see a linter step in the `build-ctst-test-image` step (I am not able to see/don't remember where we currently check that, but it should be done before any Zenko deployment), so we can quickly get notified about any error when pushing new code

Reply:
Yes, but again, I think this should be done on the last linting PR
https://github.com/scality/Zenko/pull/1821

Reply:
I would prefer having it starting this PR, and only enable the checks for the files currently linted, and then include more files in each PR, otherwise I cannot approve any PR with confidence that the linting is complete

(you can also rebase your PR to benefit from all the recent improvements in the CI, it will make your builds a bit more reliable and CTST will be way faster)